src/vm/threads.h

```diff
@@ -3721,7 +3721,7 @@ class Thread: public IUnknown
     ARG_SLOT CallPropertyGet(BinderMethodID id, OBJECTREF pObject);
     ARG_SLOT CallPropertySet(BinderMethodID id, OBJECTREF pObject, OBJECTREF pValue);
 
-#if defined(FEATURE_HIJACK) && !defined(PLATFORM_UNIX)
+#if defined(FEATURE_HIJACK) && !defined(FEATURE_PAL)
```

Comment:
Can you please revert this? FEATURE_PAL and PLATFORM_UNIX is equivalent and I'd like us to not to change stuff that can stay as it is. We have many `#if defined(FEATURE_HIJACK) && !defined(PLATFORM_UNIX)` in the code